_data/authors.json

```
@@ -87,6 +87,11 @@
         "name": "Piotr Klapczyński",
         "bio": "Piotr has been a Site Reliability Engineer for seven years. Currently, he supports stability as a SRE team leader responsible for the development of chaos engineering tools and performance tests.",
         "twitter": "klaper_",
-        "linkedin": "piotr-klapczyński-6540a63"
+        "linkedin": "piotr-klapczyński-6540a63,"
```

Tutaj wkradł się niepotrzebny przecinek :wink:
```suggestion
        "linkedin": "piotr-klapczyński-6540a63"
```